wimm.Secundatives.UnitTests/Extensions/AsMaybe_Test.cs

```diff
@@ -0,0 +1,49 @@
+﻿using wimm.Secundatives.Extensions;
+using Xunit;
+
+namespace wimm.Secundatives.UnitTests.Extensions
+{
+    public class AsMaybe_Test
+    {
+        [Fact]
+        public void AsMaybeClass_NullValue_ReturnsNone()
+        {
+            string value = null;
+
+            var res = value.AsMaybe();
+
+            Assert.Equal(Maybe<string>.None, res);
+        }
+
+        [Fact]
+        public void AsMaybeClass_Value_ReturnsSome()
+        {
+            var value = "doot";
+            var res = value.AsMaybe();
+
+            Assert.True(res.Exists);
```

Comment:
I'd test the value instead of exists since getting a value w/o exception demonstrates existence anyway